content/load-testing/Dockerfile.influxdb

```diff
@@ -0,0 +1,8 @@
+FROM influxdb:1.2.2
```

Comment:
We also support a `github-include` short code. This might be better than duplicating everything here.